apps/google-analytics-4/.gitignore

```diff
@@ -22,3 +22,6 @@ lambda/config/serverless-env.dev.yml
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# dynamodb dev
+*.db
```

Comment:
is this _too_ greedy you think? what if we want a .db file for test fixture data or sth?

Reply:
If we eventually want those than definitely too greedy, I'll change this for what dynamo automatically creates  `shared-local-instance.db`